npc/re/custom/lasagna/lasagna_npcs_2.txt

```diff
@@ -0,0 +1,2065 @@
+		mes "It takes a while to embroider the name.";
+		mes "I'll take it to the chief when it's finished, so you can go first.";
+		close;
+	}
+	if (lasagna_quest[1] == 11) {
+		mes "[Tali]";
+		mes "It takes a while to embroider the name.";
+		mes "I'll take it to the chief when it's finished, so you can go first.";
+		close;
+	}
+}
+
+lasagna,205,325,5	script	Eggling#dorcon	4_M_MERCAT1,2,2,{
+	if (lasagna_quest[1] > 0 && lasagna_quest[1] < 12) {
+		mes "[Eggling]";
+		mes "Aren't you being trained in the vigilante corps? How can you get on the ship when you are not good enough to finish the training? Go back and finish your training!";
+		close;
+	}
+	if (lasagna_quest[1] == 12) {
+		if (countitem(25046) < 1) {// Boarding Pass
+			mes "[Eggling]";
+			mes "The ticket.";
+			next;
+			mes "[" + strcharinfo(0) + "]";
+			mes "The ticket?";
+			next;
+			mes "[Eggling]";
+			mes "don't you have the ^0000cdBoarding Permission^000000? You need it to get on the ship! Hold yourself together! Go back and get one!";
+			close;
+		}
+		if (Class != Job_Summoner) {
+			mes "[Eggling]";
+			mes "The ticket.";
+			next;
+			mes "[Eggling]";
+			mes "Are you going to get on the ship?";
+			next;
+			if (select( "You have a business in the ship.", "Leave." ) == 2) {
+				mes "[Eggling]";
+				mes "Well then.";
+				close;
+			}
+			mes "[Eggling]";
+			mes "Get on now. If it is your first time, meet the purser and listen to the boarding instructions.";
+			delitem 25046,1;// Boarding Pass
+			setquest 14545;// To Purser
+			completequest 14545;
+			setquest 14531;// Everything about Con-Chliina
+			lasagna_quest[1] = 13;
+			close2;
+			warp "conch_in",58,61;
+			end;
+		}
+		mes "[Eggling]";
+		mes "The ticket.";
+		next;
+		mes "[" + strcharinfo(0) + "]";
+		mes "Here...";
+		next;
+		mes "[Eggling]";
+		mes "Hey, adventurer! Is it your first voyage?";
+		next;
+		mes "[" + strcharinfo(0) + "]";
+		mes "Yes.";
+		next;
+		mes "[Eggling]";
+		mes "Well, don't bring dishonor to the name of Doram, OK? However, since you have been trained here, you will be able to play your part wherever you are. Good luck.";
+		next;
+		mes "[" + strcharinfo(0) + "]";
+		mes "Thank you.";
+		next;
+		mes "[Eggling]";
+		mes "Oh, I have a favor to ask you. Would you bring it to the ^0000cdSous Chef^000000. Can you do this for me? You will keep sailing on this ship from time to time, you know, right?";
+		next;
+		mes "[" + strcharinfo(0) + "]";
+		mes "Yes.";
+		next;
+		mes "[Eggling]";
+		mes "Well, well. You seem to know how things work around here. You can bring it to the Sous Chef. However, the Sous Chef is kind of eccentric. You have to say something when you hand over things to him.";
+		next;
+		mes "[Eggling]";
+		mes "You should toss this to him, saying ��^0000cd[Here comes the newbie!]^000000 He believes that a Doram should say this on his or her first voyage for good luck. Anyways, he seems to think that it is the right way to do.";
```

Comment:
incorrect chars here. Encoding maybe?
looks in game weird: ![](https://i.imgur.com/fXj8yJY.png)